python/ngen_conf/src/ngen/config/init_config/soil_freeze_thaw.py

```diff
@@ -0,0 +1,192 @@
+from enum import Enum
+from typing import Any, Dict, List, Literal, Optional, Union
+
+from ngen.init_config import serializer_deserializer
+from pydantic import root_validator
+from typing_extensions import override
+
+from .utils import FloatUnitPair
+from .value_unit_pair import ListUnitPair, ValueUnitPair
+
+m = Literal["m"]
+m_per_m = Literal["m/m"]
+k = Literal["K"]
+empty = Literal[""]
+# empty string is equivalent to h / hr
+time_unit = Literal["s", "sec", "", "h", "hr", "d", "day"]
+
+
+class IceFractionScheme(str, Enum):
```

Comment:
I thought these were rainfall runoff partitioning schemes, not Ice Fraction schemes 🤯